packages/components/src/Text/Text.tsx

```diff
@@ -4,7 +4,7 @@ import { Box, system } from '@marigold/system';
 type TextProps = {};
 
 export const Text = system<TextProps, 'span'>(
-  ({ as = 'span', variant, children, ...props }) => {
+  ({ as = 'span', variant = 'body', children, ...props }) => {
```

Comment:
Why? Plus -> this really should be an extra PR.

Reply:
I thought to set a default variant. But I'll just treat the Text component in the other PR completely